incubator-tools/cer_wer/cer_wer.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>Provide more description of what these are.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Acknowledged.</p>